Mrs_Cake/Startup.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
-using Mrs_Cake.Services;
+using Mrs_Cake.MrsCakeData;
```

Comment:
@abdelilahAchir Why you changed from services to data? It means services was taking inbuilt data or data will take from databse? Please elaborate